jackal_control/launch/control.launch

```diff
@@ -8,9 +8,16 @@
         args="jackal_joint_publisher jackal_velocity_controller" />
 
   <group if="$(arg enable_ekf)" >
-    <node pkg="robot_localization" type="ekf_localization_node" name="ekf_localization">
-      <rosparam command="load" file="$(find jackal_control)/config/robot_localization.yaml" />
-    </node>
+    <group if="$(optenv JACKAL_GX5_IMU 0)">
```

Comment:
How about just loading the extra IMU into robot_localization as `imu1`?

Reply:
We'd still need to overwrite the existing imu0 configuration though.

Reply:
Discussed with Jeff; disabling fields in the internal IMU config was only needed as a work-around until the GX5 driver added support for NED/ENU frame differentiation.  That's present now, so I've changed the yaml & launch files